torch/nn/modules/module.py

```diff
@@ -618,6 +627,7 @@ def remove_from(*dicts):
                 raise AttributeError(
                     "cannot assign parameters before Module.__init__() call")
             remove_from(self.__dict__, self._buffers, self._modules)
+            self._persistent_buffers.discard(name)
```

Comment:
nit: I think you can add the discard call inside the `remove_from` function here to make this slightly simpler.

Reply:
Done.